### ISSUE TITLE
Command Line Interface (CLI) for Latest

### ISSUE THREAD

**Issue by MattiasMentzer** [mangerlahn/Latest#13]
Hey there, 

wouldn't it be nice to have a command line interface for Latest like [📦 Mac App Store command line interface](https://github.com/mas-cli/mas).
Or even a combined version of both of these helpful tools.

Keep up the good work.

**Comment by r-darwish** [comment]
https://github.com/r-darwish/topgrade/issues/657

I would like to incoroprate Latest into Topgrade, so a command line interface would be very helpful

**Comment by scriptsandthings** [comment]
Yes, being able to install this on a client, then use a script to downlod/install all updates via MDM policy would be amazing

**Comment by ammannit** [comment]
Any news?

**Comment by mangerlahn** [comment]
Unfortunately not at the moment, but I planned some restructuring that would make a CLI tool easier in the future. Though I can't promise a timeline yet :/

**Comment by sammcj** [comment]
This would be great, could then sit right next to my other brew update/upgrade commands.

**Comment by qns7** [comment]
I am also very interested in this. That's the only thing missing (together with homebrew and mas) to update ALL apps on ones machine, because latest catches the manually installed apps. :-)
